### DISCUSSION TITLE
ReferenceError: React is not defined

### DISCUSSION THREAD

**Discussion by pranavp10** [tailwindlabs/headlessui#1730] [Help] [answered]
<img width="509" alt="image" src="https://user-images.githubusercontent.com/49075129/182077574-28c0a3b8-867c-48fc-bbe2-f366700e4d1c.png">

I am using Vite instead of CRA I am getting this error in production it's due to headless UI any solution?

**Comment by baumerdev** [comment, marked as answer]
I opened an issue https://github.com/tailwindlabs/headlessui/issues/1733 for that problem.
Seems to me as if it is easily fixable in headless-ui's source code.
